### PR TITLE
changes for ansible upgrade 9.1.0

### PR DESCRIPTION
**Pull request by pvisweswar**
Staging, India and Production

https://dimagi-dev.atlassian.net/browse/SAAS-14998

### CODE REVIEW

File: src/commcare_cloud/ansible/roles/lpar2rrd/tasks/main.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
-- include: install_agent.yml
+- include_tasks: install_agent.yml
   tags: lpar2rrd_agent
-- include: install_master.yml
+- include_tasks: install_master.yml
   when: inventory_hostname in groups.lpar2rrd_master|default([])
```
Comment (pvisweswar) [inline src/commcare_cloud/ansible/roles/lpar2rrd/tasks/main.yml:5]: The first task (install_agent.yml) is a static task because it is included unconditionally and does not depend on any dynamic conditions or variables.

The second task (install_master.yml) is a dynamic task because it is included conditionally based on the value of the inventory_hostname variable. If the inventory_hostname is in the group specified by groups.lpar2rrd_master, then the task is included; otherwise, it is skipped.

Therefore, while the first task is static, the second task is dynamic.

File: src/commcare_cloud/ansible/roles/postgresql/tasks/main.yml [hunk 2]
```diff
@@ -33,7 +33,7 @@
   tags: pghashlib
 
 - name: Setup plproxy
-  include: plproxy.yml
+  include_tasks: plproxy.yml
   when: item.get('host')
   loop:
     - "{{ postgresql_dbs.form_processing.proxy|default({}) }}"
```
Comment (pvisweswar) [inline src/commcare_cloud/ansible/roles/postgresql/tasks/main.yml:39]: the tasks are dynamic because their inclusion depends on the presence of the host attribute in each item of the loop.

File: src/commcare_cloud/ansible/roles/rabbitmq/tasks/main.yml [hunk 3]
```diff
@@ -1,8 +1,8 @@
 ---
-- include: install_latest.yml
+- include_tasks: install_latest.yml
   when: rabbitmq_version == '3.10.13-1'
 
-- include: install_old.yml
+- include_tasks: install_old.yml
   when: rabbitmq_version == '3.10.7-1'
```
Comment (pvisweswar) [inline src/commcare_cloud/ansible/roles/rabbitmq/tasks/main.yml:7]: the tasks are dynamic because their inclusion depends on the value of rabbitmq_version.

File: src/commcare_cloud/ansible/roles/rabbitmq/tasks/main.yml [hunk 4]
```diff
@@ -40,19 +40,19 @@
   become: yes
   service: name=rabbitmq-server state=started
 
-- include: config-cluster.yml
+- include_tasks: config-cluster.yml
   when: rabbitmq_create_cluster|bool
 
-- include: config.yml
+- include_tasks: config.yml
   when: not rabbitmq_create_cluster or ( ansible_fqdn == hostvars[rabbitmq_cluster_master]['ansible_fqdn'] )
```
Comment (pvisweswar) [inline src/commcare_cloud/ansible/roles/rabbitmq/tasks/main.yml:47]: the tasks are dynamic because their inclusion depends on the values of rabbitmq_create_cluster, ansible_fqdn, and rabbitmq_cluster_master.

File: src/commcare_cloud/ansible/roles/rabbitmq/tasks/main.yml [hunk 5]
```diff
@@ -40,19 +40,19 @@
...
   service: name=rabbitmq-server state=started
   tags: after-reboot
 
-- include: join-cluster.yml
+- include_tasks: join-cluster.yml
   when: (rabbitmq_create_cluster | bool) and ( ansible_fqdn != hostvars[rabbitmq_cluster_master]['ansible_fqdn'] )
 
-- include: ha-policy.yml
+- include_tasks: ha-policy.yml
   when: (rabbitmq_create_cluster| bool ) and ( ansible_fqdn == hostvars[rabbitmq_cluster_master]['ansible_fqdn'] )
```
Comment (pvisweswar) [inline src/commcare_cloud/ansible/roles/rabbitmq/tasks/main.yml:58]: the tasks are dynamic because their inclusion depends on the values of rabbitmq_create_cluster, ansible_fqdn, and rabbitmq_cluster_master.

File: src/commcare_cloud/ansible/roles/redis/tasks/main.yml [hunk 6]
```diff
@@ -1,6 +1,6 @@
 ---
 
-- include: setup_cluster.yml
+- include_tasks: setup_cluster.yml
   when: is_redis_cluster|bool and inventory_hostname in groups['redis_cluster_master']
```
Comment (pvisweswar) [inline src/commcare_cloud/ansible/roles/redis/tasks/main.yml:5]: the task is dynamic because its inclusion depends on the values of is_redis_cluster and the presence of the Ansible hostname in the redis_cluster_master group.

File: src/commcare_cloud/ansible/roles/bootstrap-users/tasks/main.yml [hunk 7]
```diff
@@ -141,7 +141,7 @@
 - name: check users password valid time
   shell: getent shadow "{{ item }}" | cut -d':' -f5
   register: users_pw_valid
-  loop: "{{ dev_users.present }} + ['ansible', '{{ cchq_user }}']"
+  loop: "{{ ['ubuntu'] + ['ansible', cchq_user] }}"
   changed_when: False
```
Comment (pvisweswar) [inline src/commcare_cloud/ansible/roles/bootstrap-users/tasks/main.yml:145]: change this to loop: "{{ ['dev_users.present'] + ['ansible', cchq_user] }}"

File: src/commcare_cloud/ansible/roles/lpar2rrd/tasks/main.yml [hunk 8]
```diff
@@ -1,6 +1,6 @@
 ---
-- include: install_agent.yml
+- include_tasks: install_agent.yml
   tags: lpar2rrd_agent
```
Comment (pvisweswar) [inline src/commcare_cloud/ansible/roles/lpar2rrd/tasks/main.yml:3]: The first task (install_agent.yml) is a static task because it is included unconditionally and does not depend on any dynamic conditions or variables.

File: src/commcare_cloud/ansible/roles/bootstrap-users/tasks/main.yml [hunk 9]
```diff
@@ -141,7 +141,7 @@
 - name: check users password valid time
   shell: getent shadow "{{ item }}" | cut -d':' -f5
   register: users_pw_valid
-  loop: "{{ dev_users.present }} + ['ansible', '{{ cchq_user }}']"
+  loop: "{{ ['ubuntu'] + ['ansible', cchq_user] }}"
```
Comment (millerdev) [inline src/commcare_cloud/ansible/roles/bootstrap-users/tasks/main.yml:144]: This doesn't look like it does what it did before. Would this work instead?
```suggestion
  loop: "{{ dev_users.present + ['ansible', cchq_user] }}"
```

File: src/commcare_cloud/ansible/roles/elasticsearch/tasks/main.yml [hunk 10]
```diff
@@ -83,13 +83,13 @@
...
+- include_tasks: misc_v2.yml
   when: elasticsearch_version == '2.4.6'
 
-- include: misc_v5.yml
+- include_tasks: misc_v5.yml
   when: elasticsearch_version == '5.6.16'
 
-- include: misc_v7.yml
+- include_tasks: misc_v7.yml
   when: elasticsearch_version is version('7.0.0', '>=')
```
[the remark this answers is not in the document]
Comment (millerdev) [reply src/commcare_cloud/ansible/roles/elasticsearch/tasks/main.yml:93]: It's not totally clear whether `include_tasks` or `import_tasks` is the best replacement. The deprecated [`include` docs](https://docs.ansible.com/ansible/8/collections/ansible/builtin/include_module.html) aren't super clear on how to know which to use. I think `include_tasks` (which is dynamic) will work here and may even be required in situations where a `when` condition is applied to the include.

To summarize, this looks fine unless Ansible doesn't like it for some reason.
